src/build_scripts/buildlib.csm_share

```diff
@@ -116,7 +116,7 @@ def buildlib(bldroot, installpath, case):
         copyifnewer(_file, os.path.join(installdir, "include", os.path.basename(_file)))
 
     # This runs the make command
-    gmake_opts = "-f {}/Makefile complib MODEL=csm_share ".format(os.path.join(caseroot,"Tools"))
+    gmake_opts = "-f {}/Makefile complib MODEL=csm_share COMP_NAME=csm_share ".format(os.path.join(caseroot,"Tools"))
```

Comment:
We already have MODEL - why COMP_NAME?

Reply:
It's a clearer name. Sometimes model is E3SM/CeSM.